docs/conf.py

```diff
@@ -0,0 +1,122 @@
+"""
+Sphinx documentation generator configuration.
+
+More information on the configuration options is available at:
+
+    http://www.sphinx-doc.org/en/master/usage/configuration.html
+"""
+
+from os import environ
+from pkg_resources import get_distribution
+
+import django
+import sphinx_rtd_theme
+
+environ.setdefault("DJANGO_SETTINGS_MODULE", "test_app.settings")
+django.setup()
+
+# -- Extra custom configuration  ------------------------------------------
+
+title = "django-smart-selects documentation"
+description = ('Quickly filter or group "chained" models by adding a custom foreign key or many to many field to your models.',)
+
+# -- General configuration ------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings.
+# They can be extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = ["sphinx.ext.autodoc", "recommonmark"]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ["_templates"]
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string: source_suffix = ['.rst', '.md']
+source_suffix = ".rst"
```

Comment:
Shouldn't this be `".md"`?

Reply:
I noticed that in some documentation, but in the end I followed this [getting started guide](https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx) and it didn't mention it. It works when running `make html` so I suppose the plugin takes care of that detail.

Reply:
I tried usgin `source_suffix = ".rst"` but then it won't find `index.rst`, and we really need rst here because of the features. PR updated to make `source_suffix = ['.rst', '.md']` explicit.